macros/dbt_utils/sql/date_spine.sql

```diff
@@ -0,0 +1,46 @@
+{% macro trino__date_spine(datepart, start_date, end_date) %}
+
+
+{# call as follows:
```

Comment:
Is the comment needed?

Reply:
it comes from the original dbt implementation, I would say it's ok to keep it.